Package.swift

```diff
@@ -722,6 +722,7 @@ let package = Package(
             cxxSettings: cxxSettings,
             linkerSettings: [
                 .linkedLibrary("z"),
+                .linkedFramework("Security", .when(platforms: [.macOS, .iOS, .tvOS, .watchOS])),
```

Comment:
Does this need `.macCatalyst` too?

Reply:
Added .macCatalyst

Reply:
The reference to `.macCatalyst` needs to be guarded by a swift version check as is done for REALM_HAVE_SECURE_TRANSPORT as it doesn't exist until xcode 13.